dune-project

```diff
@@ -389,10 +389,10 @@ For more information see https://semgrep.dev
 ;TODO: restore  "bisect_ppx" {>= "2.5.0"} once can use ppxlib 0.22.0
   (depends
     ; core deps
-    (ocaml (>= 4.13.0))
+    (ocaml (>= 5.0.0))
     (dune (>= 2.7.0 ))
     ; parser generators
-    (menhir (= 20211128)) ; Newer versions cause massive build slowdowns
+    (menhir (= 20220210)) ; Newer versions cause massive build slowdowns
```

Comment:
is the comment still true? Is the new version you pinned is slow?

Reply:
I would need to investigate this, I missed it.

Reply:
It looks like compilation actually is faster...
(on 5.0)
![image](https://github.com/returntocorp/semgrep/assets/49291449/76d1235e-901f-457f-9044-f30c11684e0d)

(on `develop`)
![image](https://github.com/returntocorp/semgrep/assets/49291449/5ca6d765-8a52-46a4-8436-bf6dfa8e64fd)